frontend/src/js/query-node-editor/QueryNodeEditor.tsx

```diff
@@ -159,6 +161,37 @@ const QueryNodeEditor = ({ node, ...props }: QueryNodeEditorPropsT) => {
       ? RIGHT_SIDE_WIDTH_COMPACT
       : RIGHT_SIDE_WIDTH);
 
+  // Automatic node label generation
+  const formatConceptLabels = (labels: string[]) =>
+    labels
+      .map((label) =>
+        label
+          .split("")
+          .filter((c) => c.match(/[a-zA-Z0-9]/))
+          .join(""),
+      )
```

Comment:
you could probably just do `labels.map(label => label.match(...).join(""))`

Reply:
Yes you're right :)